replica/table.cc

```diff
@@ -493,6 +493,13 @@ compaction_group& table::compaction_group_for_sstable(const sstables::shared_sst
     return *_compaction_group;
 }
 
+std::vector<lw_shared_ptr<compaction_group>> table::compaction_groups() const {
+    // FIXME: will return table::_compaction_groups, with no allocation required, once multiple groups are supported.
```

Comment:
No allocation will be required if it will change to return `const std::vector<lw_shared_ptr<compaction_group>>&`

Reply:
That's indeed my intention. I forgot to change the function signature in the end.